### PR TITLE
Add nodejs bin path to PATH variable when installing from binary

### PR DESCRIPTION
**Pull request by rishabhtulsian**
@BarthV @guilhem @mdxp @smith - One of you kindly review and merge this pull request

### CODE REVIEW

File: recipes/nodejs_from_binary.rb [hunk 1]
```diff
@@ -57,5 +57,6 @@
   version node['nodejs']['version']
   checksum checksum
   has_binaries binaries
+  append_env_path node['nodejs']['binary']['append_env_path']
```
Comment (guilhem) [inline recipes/nodejs_from_binary.rb]: I don't understand why `has_binaries` don't do the job

Comment (guilhem) [inline recipes/nodejs_from_binary.rb]: Is there any other bin to add?

Comment (rishabhtulsian) [inline recipes/nodejs_from_binary.rb]: Without this the node binary path was not added to the PATH env variable and the 'node' command was not found

Comment (rishabhtulsian) [inline recipes/nodejs_from_binary.rb]: @guilhem Did you try it out?

Comment (sgtcoolguy) [inline recipes/nodejs_from_binary.rb]: I had this exact same issue. `append_env_path` adds the `/usr/local/nodejs-binary/bin` to `PATH`. `has_binaries` just symlinks the node and npm binaries into `/usr/local/bin`

This is a good fix  - though it brings up a bug in ark on Mac where they use root:root to add the profile.d script, and that Mac's /etc/profile doesn't use profile.d by default unless you hack it. May want to limit it to linux platform until ark is fixed to use root:wheel on mac.

**Comment by aberrios85** [comment]
LGTM? Hitting this issue on CentOS-7

**Comment by MarkGibbons** [comment]
I think this one needs an inspect test to verify the behavior.

**Comment by MarkGibbons** [comment]
#212 rebases and does this.

**Pull request closed without merging**